backend/src/main/java/ca/gov/dtsstn/cdcp/api/controller/SubscriptionController.java

```diff
@@ -0,0 +1,45 @@
+package ca.gov.dtsstn.cdcp.api.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.gov.dtsstn.cdcp.api.dto.ImmutableSubscription;
+
+@RestController
+@RequestMapping(path={"/v1/user"})
```

Comment:
REST resources are typically plural.. so `/v1/users`

Also, does it make sense to bind the *subscription* controller to the user resource? What if in the future we want to add a `/subscriptions` endpoint to get all subscriptions?

My suggestion here would be to bind to `/api/v1` as the root, rename the class to `SubscriptionControllerV1`, and then bind each endpoint as required..

``` java
@RequestMapping({ "/api/v1" })
public class SubscriptionControllerV1 {
	@GetMapping({ "/users/{userId}/subscriptions" })
	public List<Subscription> getSubscriptions(@PathVariable String userId) { .. }
}
```

Reply:
updated